AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/common.py

```diff
@@ -75,14 +75,14 @@ def set_glob_attr(nc_file_obj, data, metadata):
     :return:
     """
     setattr(nc_file_obj, 'title', metadata['title'])
-    setattr(nc_file_obj, 'site_code', metadata['site_code'])
+    # setattr(nc_file_obj, 'site_code', metadata['site_code'])
     setattr(nc_file_obj, 'site_name', metadata['site_name'])
-    setattr(nc_file_obj, 'instrument_maker', metadata['instrument_maker'])
-    setattr(nc_file_obj, 'instrument_model', metadata['instrument_model'])
-    setattr(nc_file_obj, 'waverider_type', metadata['waverider_type'])
+    setattr(nc_file_obj, 'instrument', metadata['instrument_maker'] + ' ' + metadata['instrument_model'])
```

Comment:
the input  metadata file should be updated to account for the new metadada, for example here, instead of using these 2 attributes, create a new one "instrument"

Reply:
I will find out how to do that

Reply:
you just need to update the file buoys_metadata.csv or create a new one

Reply:
easy, didn't realise they were there. That explains more stuff too related to DES-QLD